SlowCheetah.JDT/JdtExtensions.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Sayed Ibrahim Hashimi. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See  License.md file in the project root for full license information.
+
+namespace SlowCheetah.JDT
+{
+    using System;
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// Defines extension methods used in JDT
+    /// </summary>
+    internal static class JdtExtensions
+    {
+        /// <summary>
+        /// Merges an array into another.
+        /// Merges a clone of the array.
+        /// </summary>
+        /// <param name="original">Array to merge into</param>
+        /// <param name="arrayToMerge">Array to be merged</param>
+        internal static void MergeInto(this JArray original, JArray arrayToMerge)
```

Comment:
The name of this function is somewhat confusing. A.MergeInto(B) actually merge B into A. Maybe we should not use 'Into' here?